jni/Android.mk

```diff
@@ -15,11 +15,38 @@ $(warning INIPARSER SRC is going to be downloaded!)
 
 INIPARSER_ROOT :=./iniparser
 
+endif
+endif
+
 $(info $(shell ($(LOCAL_PATH)/prepare_iniparser.sh )))
 
+include $(CLEAR_VARS)
+
+NNTRAINER_JNI_ROOT := $(NNTRAINER_ROOT)/jni
+
+TENSORFLOW_VERSION := 1.13.1
+
+ifndef TENSORFLOW_ROOT
+ifneq ($(MAKECMDGOALS),clean)
+$(warning TENSORFLOW_ROOT is not defined!)
+$(warning TENSORFLOW SRC is going to be downloaded!)
```

Comment:
I think this should go to the Application or install this depending an option to use tflite backbone or not. I think we should not have tight dependency with tflite.

Reply:
if tflite backbone is enabled to be supported, then in order to compile tflite backbone layer, nntrainer will have to compile with tflite for now.
Later, we can make pluginable layers which would allow compiling layers independently and plugigng them into the nntrainer. However, this is not directly possible with current design.

Reply:
Added an option to disable tflite dependency in Android.mk for android build.
I am not able to read this option from meson_options.txt
Please let me know if you know how to.